Framework/Kernel/inc/MantidKernel/V2D.h

```diff
@@ -88,6 +88,13 @@ class MANTID_KERNEL_DLL V2D {
     Y() *= factor;
     return *this;
   }
+  ///  Divide by d
+  inline V2D operator/(const double d) const noexcept { return V2D(X() / d, Y() / d); }
```

Comment:
Looks like this division operation is something we want all the `VMD` objects to support (I think `V3D` already does...), so maybe needs some abstract method in `VMD` that needs to be implemented?

Reply:
I hadn't come across `VMD` before but seems like `V2D` and `V3D` are standalone classes and don't inherit from `VMD` (or `VMDBase`) so don't think I can put in a pure virtual method on any base class here - at least not without messing with the `V2D` and `V3D` class hierarchy which is perhaps a bit much for this PR

Reply:
OK thanks for checking I thought they did! I think no need to go messing with these data types in this PR then!